crates/relayer-cli/src/chain_registry.rs

```diff
@@ -57,7 +57,12 @@ fn construct_packet_filters(ibc_paths: Vec<IBCPath>) -> HashMap<String, PacketFi
 
     packet_filters
         .into_iter()
-        .map(|(k, v)| (k, PacketFilter::Allow(ChannelFilters::new(v))))
+        .map(|(k, v)| {
+            (
+                k,
+                PacketFilter::new(ChannelPolicy::Allow(ChannelFilters::new(v)), HashMap::new()),
```

Comment:
Let's add a smart constructor to `PacketFilter` to turn this into

```suggestion
                PacketFilter::allow(v),
```